lib/rubygems/commands/yank_command.rb

```diff
@@ -61,15 +62,23 @@ def execute
 
   def yank_gem(version, platform)
     say "Yanking gem from #{self.host}..."
-    yank_api_request(:delete, version, platform, "api/v1/gems/yank")
+    args = [:delete, version, platform, "api/v1/gems/yank"]
+    response = yank_api_request(*args)
```

Comment:
Why are we not passing `options[:otp]` here? I don't think we need `use_otp` argument. If `options[:otp]` is empty/nil, don't add it to request.
Please fix this for other commands as well.